test/fixtures/dialects/snowflake/snowflake_create_clone.sql

```diff
@@ -0,0 +1,13 @@
+-- Taken from Snowflake Documentation https://docs.snowflake.com/en/sql-reference/sql/create-clone.html
+
+CREATE DATABASE mytestdb_clone CLONE mytestdb;
```

Comment:
Next time, break two logical changes into two PRs (one for Snowflake clone, one for Postgres view options).

Reply:
Apologies wasn't sure if it was ok to group them

Reply:
But good to have coverage for this though, thanks!